### DISCUSSION TITLE
How to run project locally?

### DISCUSSION THREAD

**Discussion by itzik-bd** [ben-gibson/GitLink#263] [General]
Hi there, really liked this plugin, and wanted to contribute 👨‍💻 

After cloning the repo, I can't download `uk.co.ben_gibson:url:0.0.9` artifact via gradle:
```
Could not GET 'https://maven.pkg.github.com/ben-gibson/url/uk/co/ben_gibson/url/0.0.9/url-0.0.9.pom'. Received status code 401 from server: Unauthorized
```

Saw that in [build.gradle.kts](https://github.com/itzik-bd/GitLink/blob/6cfac49a6102f4c9d4ac162fe3770603b7cd561a/build.gradle.kts#L27-L30), it uses GitHub credentials, so tried to create personal access token, but it still get the `Unauthorized` error.

Any tip here would be highly appreciate! 🙏

**Comment by ben-gibson** [comment]
Yeah, I was attempting to publish the `url` library, but it turned out to be more involved than I anticipated. The package repository on GitHub was easy to use, but it doesn't seem to be public. 

If you still wanted to contribute, you could clone the url library, build the jar, and just copy it into `GitLink` instead, then remove the dependency from maven altogether. That would solve a bunch of other problems too.

**Reply by itzik-bd** [reply]
Thanks! will try in soon 🙏
